tests/data/condor_submit_dag/dagbegin.cmd

```diff
@@ -0,0 +1,66 @@
+# generated by jobsub_lite
+# debug
+universe           = vanilla
+executable         = sambegin.sh
+arguments          =
+
+
+output             = sambegin.$(Cluster).$(Process).out
+error              = sambegin.$(Cluster).$(Process).err
+log                = sambegin.$(Cluster).$(Process).log
+environment        = CLUSTER=$(Cluster);PROCESS=$(Process);CONDOR_TMP=/home/$ENV(USER)/.cache/jobsub_lite/js_2023_12_18_112255_71a1aea7-4417-446c-920c-d3042a8f2b4b;BEARER_TOKEN_FILE=.condor_creds/fermilab.use;CONDOR_EXEC=/tmp;DAGMANJOBID=$(DAGManJobId);GRID_USER=$ENV(USER);JOBSUBJOBID=$(CLUSTER).$(PROCESS)@jobsubdevgpvm01.fnal.gov;EXPERIMENT=fermilab;EXPERIMENT=samdev;IFDH_DEBUG=1;IFDH_VERSION=v2_6_10;IFDH_TOKEN_ENABLE=1;IFDH_PROXY_ENABLE=0;SAM_EXPERIMENT=samdev;SAM_GROUP=samdev;SAM_STATION=samdev;IFDH_CP_MAXRETRIES=2;VERSION=v1_1;SAM_DATASET=gen_cfg;SAM_USER=$ENV(USER);SAM_PROJECT=$ENV(SAM_PROJECT)
+rank                  = Mips / 2 + Memory
+notification  = Error
++RUN_ON_HEADNODE= True
+rank               = Mips / 2 + Memory
+job_lease_duration = 3600
+transfer_error     = True
+transfer_executable= True
+when_to_transfer_output = ON_EXIT_OR_EVICT
+transfer_output_files = .empty_file
+request_memory = 100mb
+
++JobsubClientDN="/DC=org/DC=cilogon/C=US/O=Fermi National Accelerator Laboratory/OU=People/CN=Marc Mengel/CN=UID:$ENV(USER)/CN=3752395701"
```

Comment:
Is there a way of not having this DN here (or anonymizing it), or will that cause the job to fail?  Given the `CN` bit at the end, I'm assuming it should be OK to make it be a random string, but please let me know if that's wrong.

Reply:
Sigh.  Yes.   MOdifying the test to read the whole DN out of the proxy and put it in $DN in the environment....